package/system/urandom-seed/Makefile

```diff
@@ -2,10 +2,10 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=urandom-seed
 PKG_VERSION:=1.0
-PKG_RELEASE:=1
-PKG_LICENSE:=GPL-2.0
+PKG_RELEASE:=2
 
-PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+PKG_MAINTAINER:=
+PKG_LICENSE:=GPL-2.0-or-later
```

Comment:
If you change the license, you need explicit Acked-by of all previous authors.

Reply:
Hmm, it's just a renaming, it refers the same license IMO.
The only contributor of urandom-seed Makefile is ynezz, but I'm not sure if we should include the list of contributors of base-files here...

Reply:
@adrianschmutzler Where is the `LICENSE` file, hm? So it can be used variable `PKG_LICENSE_FILES`. Otherwise as said earlier current "GPL-2.0" is wrong.
Just take a look on [SPDX License list](https://spdx.org/licenses/).

Reply:
Oh, I was wrong. The list says that newer version of GPL-2.0 is GPL-2.0-only. Sorry for my confusion.